Description: Fixes to neutron configuration for distribution.
 Patch various bits of config to be more suitable for packaging.
Author: James Page <james.page@ubuntu.com>
Forwarded: not-needed

--- a/etc/neutron.conf
+++ b/etc/neutron.conf
@@ -7,7 +7,7 @@
 
 # Where to store Neutron state files.  This directory must be writable by the
 # user executing the agent.
-# state_path = /var/lib/neutron
+state_path = /var/lib/neutron
 
 # Where to store lock files
 lock_path = $state_path/lock
@@ -50,7 +50,7 @@ lock_path = $state_path/lock
 # previous versions, the class name of a plugin can be specified instead of its
 # entrypoint name.
 #
-# core_plugin =
+core_plugin = neutron.plugins.ml2.plugin.Ml2Plugin
 # Example: core_plugin = ml2
 
 # (ListOpt) List of service plugin entrypoints to be loaded from the
@@ -380,7 +380,7 @@ notification_driver = neutron.openstack.
 # Use "sudo neutron-rootwrap /etc/neutron/rootwrap.conf" to use the real
 # root filter facility.
 # Change to "sudo" to skip the filtering and just run the comand directly
-# root_helper = sudo
+root_helper = sudo /usr/bin/neutron-rootwrap /etc/neutron/rootwrap.conf
 
 # =========== items for agent management extension =============
 # seconds between nodes reporting state to server; should be less than
@@ -404,7 +404,7 @@ signing_dir = $state_path/keystone-signi
 # connection = mysql://root:pass@127.0.0.1:3306/neutron
 # Replace 127.0.0.1 above with the IP address of the database used by the
 # main neutron server. (Leave it as is if the database runs on this host.)
-# connection = sqlite://
+connection = sqlite:////var/lib/neutron/neutron.sqlite
 
 # The SQLAlchemy connection string used to connect to the slave database
 # slave_connection =
